writehat/static/js/global.js

```diff
@@ -310,6 +310,16 @@ function loadToolTips() {
       tooltipSelectModal.find('.modal-body').html(selectedTooltipText);
     });
   });
+  $('.tooltipTodo').click(function(e){
+    var componentId = $(e.currentTarget).attr('id').replace("tooltipTodo-", "");
+    loadModal('tooltipTodo', function(tooltipTodoModal) {
```

Comment:
How feasible do you think it would be to also display the todo note alongside the name of the field the todo is contained in? If this is possible, I think this would be helpful for giving a broad overview of the todo details from the report components page instead of having to click into the component to check.

![image](https://github.com/blacklanternsecurity/writehat/assets/5758342/b63cf37f-23a3-40cb-a1c3-fe1cd20469b7)

Reply:
Since the current implementation only shows whether a field contains a `{todo}` tag or not, listing the notes from the tags would require some re-engineering. 

This would require several helper methods since some of the functions rely on the list aligning with properties on the component itself, while others have to be rendered in Jinja templates which imposes syntactic limitations.

I agree this would be valuable; I considered it early on but elected to wait and see how the feature is used in practice before making such an involved change.

Reply:
gotcha, that makes sense. I say we come back to that idea in the future then, depending on how the feature is used/received.